packages/communication/.gitignore

```diff
@@ -0,0 +1,4 @@
+lib/
```

Comment:
Could this go under `packages/kits/*` or maybe `packages/sdk/*` so we can start structuring that subfolder as a workspace?

Reply:
Moved to `sdk`
and all the wallets to `sdk/wallets`